packages/codegen-ui-react/lib/__tests__/studio-ui-codegen-react-views.test.ts

```diff
@@ -26,3 +26,31 @@ describe('amplify view renderer tests', () => {
     expect(tableElement).toMatchSnapshot();
   });
 });
+
+describe('amplify view renderer tests', () => {
+  test('should render view with passed in predicate and sort', () => {
+    const { componentText, declaration } = renderWithAmplifyViewRenderer(
+      'views/post-table-datastore',
+      'datastore/post-ds',
+    );
+    expect(componentText).toContain('useDataStoreBinding');
+    expect(componentText).toMatchSnapshot();
+    expect(declaration).toMatchSnapshot();
+  });
+  test('should render view with custom datastore', () => {
+    const { componentText, declaration } = renderWithAmplifyViewRenderer('views/table-from-custom-json', undefined);
+    expect(componentText).not.toContain('useDataStoreBinding');
+    expect(componentText).toMatchSnapshot();
+    expect(declaration).toMatchSnapshot();
+  });
+
+  test('should call util file if rendered', () => {
+    const { componentText, declaration } = renderWithAmplifyViewRenderer(
+      'views/post-table-custom-format',
+      'datastore/post-ds',
+    );
+    expect(componentText).toContain('useDataStoreBinding');
```

Comment:
should we be checking if the text has a util import instead?

Reply:
ah yea can add that check in